src/PanelTraits/Fields.php

```diff
@@ -11,60 +11,55 @@
     /**
      * Add a field to the create/update form or both.
      *
-     * @param string   $form    The form to add the field to (create/update/both)
+     * @param string|array $field The new field.
+     * @param string $form The CRUD form. Can be 'create', 'update' or 'both'. Default is 'both'.
      */
     public function addField($field, $form = 'both')
     {
         // if the field_definition_array array is a string, it means the programmer was lazy and has only passed the name
         // set some default values, so the field will still work
         if (is_string($field)) {
-            $complete_field_array['name'] = $field;
+            $completeFieldsArray['name'] = $field;
```

Comment:
Is that the field definition array (i.e. is that variable oddly named)?

Reply:
Yeah. I just changed the casing. Any suggestions for a better name?

Reply:
`$allTheFields` (does it need the hungarian notiation)?
`$fieldsInformation`